packages/repl/src/lib/index.svelte

```diff
@@ -175,6 +182,9 @@
 
 		register_module_editor(editor) {
 			module_editor = editor;
+			module_editor.cursorIndex.subscribe((index) => {
+				cursor_index.set(index);
+			});
```

Comment:
I'm subscribing it here without unsubscribe since there isn't another nice way to transfer the store state to `cursor_index`. It should be safe too since codemirror also has stuff that aren't unsubscribed.